app/config/ApplicationGlobal.scala

```diff
@@ -16,53 +16,53 @@
 
 package config
```

Comment:
Delete this file and only create a MyErrorHandler without injecting things that we don't need. Look in cbcr-frontend